react/features/app/actions.js

```diff
@@ -71,12 +78,15 @@ function _appNavigateToMandatoryLocation(
      * @returns {void}
      */
     function loadConfigSettled(error, config) {
-        // FIXME Due to the asynchronous nature of the loading, the specified
-        // config may or may not be required by the time the notification
-        // arrives.
+        const { pendingLocationURL } = getState()['features/base/connection'];
+
+        if (pendingLocationURL !== locationURL) {
+            throw new Error(
```

Comment:
Pawel and I talked in person, here're my initial comments for the record:
1. loadConfigError needs to reach the external API/the SDK consumer.
2. We already have CONFIG_WILL_LOAD, LOAD_CONFIG_ERROR, and SET_CONFIG so we have all the actions describing the loading process before it starts and after it ends. Instead of introducing a new redux action (in the already complex appNavigate procedure which we plan to modify anyway), let's modify the existing actions' payloads to suit our purposes. For example, if we use a URL instance, the difference in the instances could be used as an indicator of the request instance and signal whether an action is relevant to the current redux state, reducer, middleware, etc or not.